src/pages/Landing/index.tsx

```diff
@@ -39,10 +39,17 @@ export default () => {
   // const queryClient = useQueryClient();
 
   // Queries
-  const { isLoading, data } = useQuery(
+  const { isLoading, data, error } = useQuery(
     'customers',
     getCustomers,
-    // TODO: Stretch - Use the options object to handle errors.
+    {
+      onError: (err) => {
+        console.error('Error occurred while fetching customers:', err);
+      },
+      retry: 3,
+      refetchOnWindowFocus: false,
```

Comment:
i'm curious, what drew you to add ` refetchOnWindowFocus: false,` ?

Reply:
i needed something to disable automatic refetching

Reply:
why? was it causing a problem?